alpha/hal.cpp

```diff
@@ -269,6 +338,7 @@ void _TS_HAL::btn_init()
   #ifdef HAL_M5STICK_C
   // To read interrupts from AXP192
   M5.MPU6886.setIntActiveLow();
+  M5.I2C.writeByte(0x34, 0x42, 0x03);
```

Comment:
What is this for ?

Reply:
please add comments on what those 3 values are for. Not good to hardcode addresses

Reply:
i think francis did this to make the power button work

Reply:
Yes sorry, it was just a stop gap, writing functions like these are easier to maintain than implementing them in the M5stickC library